contracts/FeeCollector.sol

```diff
@@ -45,18 +37,18 @@ contract FeeCollector is
 
     /// @inheritdoc IFeeCollector
     function distributeFees(address asset) public override onlyOwner returns (uint256 defaultFundsDelta, uint256 protocolFeesDelta) {
-        // todo: check for roudups/downs of division
         uint256 contractBalance = IERC20Minimal(asset).balanceOf(address(this));
+        uint256 distributionAmount = contractBalance > (protocolFees[asset] + defaultFund[asset]) ?
```

Comment:
contract balance should always be equal to protocol fees + default fund, right? therefore, function should revert if this is not the case, so I would just do `contractBalance - protocol fees - default fund`.